arrow/arrow.py

```diff
@@ -575,6 +594,32 @@ def float_timestamp(self):
 
         return self.timestamp + float(self.microsecond) / 1000000
 
+    @property
+    def fold(self):
+        """ Gets the ``fold`` value of the :class:`Arrow <arrow.arrow.Arrow>` object. """
+
+        # in python < 3.6 _datetime will be a _DatetimeWithFold if fold=1 and a datetime with no fold attribute
+        # otherwise, so we need to return zero to cover the latter case
+        return getattr(self._datetime, "fold", 0)
+
+    @fold.setter
+    def fold(self, val):
+        """ Sets the ``fold`` value of the :class:`Arrow <arrow.arrow.Arrow>` object. """
+
+        if val not in {0, 1}:
+            raise ValueError("fold attribute must be either 0 or 1")
+        if not dateutil_tz.datetime_ambiguous(self._datetime):
+            # for non ambiguous datetimes fold must be 0
+            self._datetime = dateutil_tz.enfold(self._datetime, fold=0)
+        else:
+            self._datetime = dateutil_tz.enfold(self._datetime, fold=val)
+
+    @property
+    def ambiguous(self):
```

Comment:
I think this is fine I suppose, but just want to make sure you realize this is not required for PEP 495.

Reply:
Yeah I think it's quite handy to have it immediately accessible.